### ISSUE TITLE
ID 258562: [Supporting the platform - Simon - Save]: Control present under 'Save dashboard' blade is getting truncated after setting the view port at 320*256.

### ISSUE THREAD

**Issue by eledobleefe** [grafana/grafana#66490]
### **User Experience:** 
Low vision who depends on zooming the page for better readability will be impacted as they will be able to see the complete content present in the page which will result in lack of understandability and will fail to access the controls. 
_Note: User credentials should NOT be included in the bug._

### **Pre-requisite:**  
Achieve viewport 320 X 256 by adjusting the browser. (Open developer's tool, resize and zoom the browser to achieve viewport 320 X 256).

### **Repro Steps:**

1. Open #Url: [play.grafana.org](https://play.grafana.org/). Sign in with valid credentials. "Home" page appears. 
2. Navigate the Feature Showcase pane heading and select it. 
3. Navigate the expanded Dropdown and select the "Edit" menuitem. 
4. Navigate the page "Edit Panel" page which appears. 
5. Navigate to "Save" button and select it. Verify the issue.

**Actual Result:** 
Control i.e. (Changes 28) present under 'Save dashboard' blade is getting truncated after setting the view port at 320*256.

**Expected Result:**
The control present under 'Save dashboard' blade should not get truncated after setting the view port at 320*256. Control should be clearly visible.

### **Severity:**
`SEV2`

### **Area:**
- [ ] **#grafana-frontend-platform**: `TabsBar Component` (Saga Design System)
_Note: Same issue exists with the controls present under 'More' menu item (Feature showcases>More)._
- [ ] **#grafana-dashboards**: `PanelHeaderMenu Component`
